scripts/docker-integration-tests/coordinator_config_rules/test.sh

```diff
@@ -0,0 +1,196 @@
+
+  echo "Test write with mapping rule"
+  label0_name="app" label0_value="nginx_edge" \
+    prometheus_remote_write \
+    foo_metric $now_truncated 42.42 \
+    true "Expected request to succeed" \
+    200 "Expected request to return status code 200"
+  label0_name="app" label0_value="nginx_edge" \
+    prometheus_remote_write \
+    foo_metric $now_truncated_plus_second 84.84 \
+    true "Expected request to succeed" \
+    200 "Expected request to return status code 200"
+
+  start=$(expr $(date +"%s") - 3600)
+  end=$(expr $(date +"%s"))
+  step="30s"
+  params_range="start=${start}"'&'"end=${end}"'&'"step=30s"
+  jq_path=".data.result[0].values | .[][1] | select(. != null)"
+
+  # Test values can be mapped to 5s:10h resolution namespace (for app="nginx")
+  echo "Test query mapping rule"
+  ATTEMPTS=50 TIMEOUT=2 MAX_TIMEOUT=4 \
+    endpoint=query_range query=foo_metric params="$params_range" \
+    jq_path="$jq_path" expected_value="84.84" \
+    metrics_type="aggregated" metrics_storage_policy="5s:10h" \
+    retry_with_backoff prometheus_query_native
+}
+
+function test_query_rollup_rule {
+  if [ "$TEST_ROLLUP_RULE" != "true" ]; then
+    echo "Skip testing rollup rule, timestamped metrics don't work with rollup rules just yet"
+    return 0
+  fi
+
+  now=$(date +"%s")
+  now_truncate_by=$(expr $now % 5)
+  now_truncated=$(expr $now - $now_truncate_by)
+  now_truncated_plus_second=$(expr $now_truncated + 1)
+
+  echo "Test write with rollup rule"
+
+  # Emit values for endpoint /foo/bar (to ensure right values aggregated)
+  label0_name="app" label0_value="nginx_edge" \
+    label1_name="status_code" label1_value="500" \
+    label2_name="endpoint" label2_value="/foo/bar" \
+    prometheus_remote_write \
+    http_requests $now_truncated 42 \
+    true "Expected request to succeed" \
+    200 "Expected request to return status code 200"
+  label0_name="app" label0_value="nginx_edge" \
+    label1_name="status_code" label1_value="500" \
+    label2_name="endpoint" label2_value="/foo/bar" \
+    prometheus_remote_write \
+    http_requests $now_truncated_plus_second 64 \
+    true "Expected request to succeed" \
+    200 "Expected request to return status code 200"
+
+  # Emit values for endpoint /foo/baz (to ensure right values aggregated)
+  label0_name="app" label0_value="nginx_edge" \
+    label1_name="status_code" label1_value="500" \
+    label2_name="endpoint" label2_value="/foo/baz" \
+    prometheus_remote_write \
+    http_requests $now_truncated 8 \
+    true "Expected request to succeed" \
+    200 "Expected request to return status code 200"
+  label0_name="app" label0_value="nginx_edge" \
+    label1_name="status_code" label1_value="500" \
+    label2_name="endpoint" label2_value="/foo/baz" \
+    prometheus_remote_write \
+    http_requests $now_truncated_plus_second 12 \
+    true "Expected request to succeed" \
+    200 "Expected request to return status code 200"
+
+  start=$(expr $(date +"%s") - 3600)
```

Comment:
nit: define `end` first, then subtract from that, just to avoid any potential issues with crossing any boundaries?